sites/svelte.dev/static/robots.txt

```diff
@@ -1,2 +1,2 @@
 User-agent: *
-Disallow: /tutorial/* # new tutorial is at learn.svelte.dev
+Disallow:
```

Comment:
Can this line be removed altogether?

Reply:
I don't know. This is what the example getting started one shows and what we have on kit.svelte.dev. I'll keep it since it seems to work